<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraControlWidget</class>
 <widget class="QWidget" name="CameraControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>203</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>203</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>203</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="serial_label">
     <property name="text">
      <string>Serial Number: xxxxxxxx</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="arm_pushButton">
     <property name="text">
      <string>Arm Camera</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="exposure_pushButton">
     <property name="text">
      <string>Update Exposure Time</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPushButton" name="start_pushButton">
     <property name="text">
      <string>Start Camera</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QRadioButton" name="video_radioButton">
     <property name="text">
      <string>Absorption</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QRadioButton" name="absorption_radioButton">
     <property name="text">
      <string>Video</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QFormLayout" name="exposure_formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="exposure_label">
       <property name="text">
        <string>Exposure Time (ms)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="exposure_lineEdit">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
